MPU/src/main.cpp

```diff
@@ -3,13 +3,18 @@
  * @author Nick DePatie
  * @brief Entry Point for Program
  */
+
 #include <nerduino.h>
 #include "mpu.h"
+#include "mpuConfig.h"
+
+int driveState;
 
 void setup()
 {
     NERduino.begin();
-    initializeCAN(1);           //The "1" parameter is useless for now, in the future the parameter is which CAN line to initialize
+    driveState = OFF;
+    initializeCAN(CANLINE_1, BAUD_RATE, &(*mpuCanCallback));
```

Comment:
Is `&(*mpuCanCallback)` always how we pass the callback function?

Reply:
Yeah this is the new way of doing it with the updated CAN framework, it works well and is tested